style.css

```diff
@@ -1,7 +1,121 @@
-h1 {
-  font-family: 'Roboto', sans-serif;
+body {
+  margin: 0;
 }
 
-span {
-  font-family: 'PT Serif', serif;
+.wrapper {
+  margin: auto;
+  width: 80%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6206393)